.github/workflows/build-push.yml

```diff
@@ -0,0 +1,84 @@
+# Name of the workflow
+name: build-push
+
+# This workflow will run when there is a push (including merge) to the listed branches
+on:
+  push:
+    branches:
+      - master
+      - v2.87
```

Comment:
we also need to add in branch `1.110` as well.

Reply:
also add it to the `run-tests.yml` pipeline as well.

Reply:
I don't see a 1.110 branch. Is it named v1.110 or just 1.110?